### PR TITLE
Run tests against Python 3.9 and Django 3.2 LTS

### PR DESCRIPTION
**Pull request by michael-k**
*Issue #, if available:* n/a

*Description of changes:*

- Run tests against Python 3.9 and Django 3.2 (the latest LTS release)
- Fix tests on Python 3.4

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -2,9 +2,10 @@
 envlist =
     py{27,34,35,36,37,38}-default
```
Comment (srprash) [inline tox.ini:3]: Should we add py39 to this and below test envs as well?

File: tox.ini [hunk 2]
```diff
@@ -35,20 +37,23 @@ deps =
...
-    py{35,36,37,38}: aiohttp >= 3.0.0
-    py{35,36,37,38}: pytest-aiohttp
-    py{35,36,37,38}: aiobotocore >= 0.10.0
+    py{35,36,37,38,39}: aiohttp >= 3.0.0
+    py{35,36,37,38,39}: pytest-aiohttp
+    py{35,36,37,38,39}: aiobotocore >= 0.10.0
 
 commands =
     py{27,34}-default: coverage run --source aws_xray_sdk -m py.test tests --ignore tests/ext/aiohttp --ignore tests/ext/aiobotocore --ignore tests/ext/django --ignore tests/test_async_local_storage.py --ignore tests/test_async_recorder.py
     py{35,36,37,38}-default: coverage run --source aws_xray_sdk -m py.test --ignore tests/ext/django tests
```
Comment (srprash) [inline tox.ini:55]: Add py39 here too?

**Review by srprash — COMMENTED** [review]
Hi @michael-k 
Thanks for adding python 3.9 to the testing suite. Can you add it to some of the missed spots as well?

File: tox.ini (hunk 1 above)
Comment (michael-k) [reply tox.ini]: Done

File: tox.ini (hunk 2 above)
Comment (michael-k) [reply tox.ini]: Done

**Review by srprash — APPROVED** [review]
Thanks!

**Pull request merged**